apps/network/app.js

```diff
@@ -326,6 +328,14 @@ module.exports = function (app) {
     })
   }
 
+  function sendSn (sn) {
+    logger.log('send SN to App: ', sn)
```

Comment:
Use string template? 
```suggestion
    logger.log(`send SN to App: ${sn}`)
```

Reply:
BTW, I think the call to `property.get` should be here, because your function name is send **SN** :)